src/containers/Layout/Layout.tsx

```diff
@@ -43,7 +52,9 @@ const Layout = () => {
     shallow,
   );
   const { updateSearchQuery, resetSearchQuery } = useSearchQueryUpdater();
-  const isLoggedIn = !!useAccountStore((state) => state.user);
+  const user = useAccountStore(({ user }) => user);
```

Comment:
I guess this can be one liner as previously.. not seeing any benefit of splitting it